.github/workflows/build.yml

```diff
@@ -27,6 +26,8 @@ jobs:
       - name: build and test
         id: thebuild
         run: ./gradlew build --info --init-script init.gradle
+      - name: integration tests
```

Comment:
Should there be a second step that runs on latest image?

Reply:
Yes, but the "ready database" is currently only available for the latest-dev and eventually next schema. Install scripts required changes for the Oracle Free database.